README.md

```diff
@@ -598,31 +734,54 @@ The following constructs are fully supported:
 * lambda expressions
 * `!` and `~` patterns (silently but successfully ignored during promotion)
 * class and instance declarations
-* scoped type variables
-* signatures (e.g., `(x :: Maybe a)`) in expressions and patterns
+* signatures (e.g., `(x :: Maybe a)`) in expressions
 * `InstanceSigs`
-* higher-kinded type variables (see below)
-* finite arithmetic sequences (see below)
-* records (with limitations -- see below)
-* functional dependencies (with limitations -- see below)
-* type families (with limitations -- see below)
 
-`singletons` is slightly more conservative with respect to `deriving` than GHC is.
-The stock classes listed above (`Eq`, `Ord`, `Show`, `Bounded`, `Enum`, `Functor`,
-`Foldable`, and `Traversable`) are the only ones that `singletons` will derive
-without an explicit deriving strategy. To do anything more exotic, one must
-explicitly indicate one's intentions by using the `DerivingStrategies` extension.
+## Partial support
+
+The following constructs are partially supported:
 
-`singletons` fully supports the `anyclass` strategy as well as the `stock` strategy
-(at least, for the classes listed above). `singletons` does not support the
-`newtype` strategy, as there is not an equivalent of `coerce` at the type level.
+* `deriving`
+* higher-kinded type variables
+* finite arithmetic sequences
+* records
+* signatures (e.g., `(x :: Maybe a)`) in patterns
+* functional dependencies
+* type families
```

Comment:
There is no follow-on section to this.

Reply:
Similarly, I think this is a reference to the "Known bugs" section, which has a bullet point about type families. I have moved this bullet point to the "Partial support" section.